challenges/python/main.py

```diff
@@ -0,0 +1,8 @@
+from test import isPalyndrome
+
+for line in open('../palindrome', 'r'):
+    str_to_test, _is_a_palindrome = line.split('\t')
+    is_a_palindrome = _is_a_palindrome == 'T'
+
+    if isPalyndrome(str_to_test) != is_a_palindrome:
+        raise ValueError("'{}' is {}a palindrome.", str, '' if is_a_palindrome else 'not ')
```

Comment:
we would like to have a full log of which tests are OK, which aren't. TO have a nice stdout (that's why we use mocha framework for js side)